app/models/worldwide_organisation.rb

```diff
@@ -72,6 +72,10 @@ def sponsoring_organisation
              content: :summary,
              format: "worldwide_organisation"
 
+  def title
+    name
+  end
```

Comment:
Could this be done with `alias_method`?  E.g. `alias_method :title :name`.

Reply:
I'm going to leave this as is because the `display_name` method below follows the same format